sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/AccessControlClientLiveTests.cs

```diff
@@ -62,7 +62,9 @@ public async Task TestGetRoleAssignment()
         public async Task TestCreateAndDeleteRoleAssignment()
         {
             string sqlAdminRoleId = "7af0c69a-a548-47d6-aea3-d00e69bd83aa";
-            string principalId = Guid.NewGuid().ToString();
+
+            // This is a random GUID for testing.
+            string principalId = "1d271944-6c4b-4464-b6ef-c77eb8bd03a9";
```

Comment:
I think it's fine to hard code this since you've already got it working and recorded.  In the future you can also use `Recording.Random.NewGuid().ToString()` if you want "random" GUIDs that are predictably random on playback _(we stash the random seed in the recording so all your calls to `Recording.Random.*` will match the recorded values during playback)_.

Reply:
Got it! Thanks. This is exactly what I want: `Recording.Random.NewGuid().ToString()`. I will update the code.